src/lib.rs

```diff
@@ -9,6 +9,25 @@
 
 //! A simple, safe HTTP client.
 //!
+//!
+//!<div align="center">
+//!  <!-- Version -->
+//!  <a href="https://crates.io/crates/ureq">
+//!    <img src="https://img.shields.io/crates/v/ureq.svg?style=flat-square"
+//!    alt="Crates.io version" />
+//!  </a>
+//!  <!-- Docs -->
+//!  <a href="https://docs.rs/ureq">
+//!    <img src="https://img.shields.io/badge/docs-latest-blue.svg?style=flat-square"
+//!      alt="docs.rs docs" />
+//!  </a>
+//!  <!-- Downloads -->
+//!  <a href="https://crates.io/crates/ureq">
+//!    <img src="https://img.shields.io/crates/d/ureq.svg?style=flat-square"
+//!      alt="Download" />
```

Comment:
Should this alt-text say "Crates.io downloads" or something like that?

Reply:
Good suggestion, thank you! I updated the alt text and also slightly changed the badge positions to show up at the top. In my opinion this looks a bit cleaner.